charts/probate-caveats-fe/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart for the HMCTS probate product
 name: probate-caveats-fe
 home: https://github.com/hmcts/probate-caveats-frontend
-version: 1.1.12
+version: 1.1.13
```

Comment:
Make any changes to the `/charts` folder in a separate PR, get that approved and merged first.
This is to avoid race conditions in Flux.